pyproject.toml

```diff
@@ -29,10 +50,72 @@ opensearch2 = [
     "invenio-search[opensearch2]>=3.0.0,<4.0.0",
 ]
 
-[tool.setuptools.dynamic]
-version = {attr = "invenio_collections.__version__"}
-readme = {file = ["README.rst"]}
+# Entrypoints
+[project.entry-points."invenio_base.apps"]
+invenio_collections = "invenio_collections.ext:InvenioCollections"
+
+[project.entry-points."invenio_base.api_apps"]
+invenio_collections = "invenio_collections.ext:InvenioCollections"
+
+[project.entry-points."invenio_base.api_blueprints"]
+invenio_collections = "invenio_collections.views:create_bp"
+
+[project.entry-points."invenio_celery.tasks"]
+invenio_collections = "invenio_collections.tasks"
+
+[project.entry-points."invenio_db.alembic"]
+invenio_collections = "invenio_collections:alembic"
+
+[project.entry-points."invenio_db.models"]
+invenio_collections = "invenio_collections.models"
+
+# Tools in alphabetical order
+[tool.babel.compile_catalog]
+directory = "invenio_collections/translations/"
+use_fuzzy = true
+
+[tool.babel.extract_messages]
+copyright_holder = "CERN"
+msgid_bugs_address = "info@inveniosoftware.org"
+mapping_file = "babel.ini"
+output_file = "invenio_collections/translations/messages.pot"
+add_comments = "NOTE"
+
+[tool.babel.init_catalog]
+input_file = "invenio_collections/translations/messages.pot"
+output_dir = "invenio_collections/translations/"
+
+[tool.babel.javascript]
+"**.js" = { encoding = "utf-8", extract_messages = ["$_", "jQuery._"] }
+
+[tool.babel.jinja2]
+"**/templates/**.html" = { encoding = "utf-8" }
+
+[tool.babel.python]
+"**.py" = { encoding = "utf-8" }
+
+[tool.babel.update_catalog]
+input_file = "invenio_collections/translations/messages.pot"
+output_dir = "invenio_collections/translations/"
+
+[tool.hatch.version]
+path = "invenio_collections/__init__.py"
+
+[tool.hatch.build]
+exclude = [
+  "run-tests.sh",
+  "tests/**",
+  "tests/docs",
```

Comment:
```suggestion
```

minor: was that a folder that we have in repos in general?

Reply:
This should be just `docs/**`, WDYT?